charactersheet/charactersheet/index.html

```diff
@@ -104,6 +133,11 @@
                 <i class="small underline"><a class="" data-toggle="modal"
                   data-target="#charactersModal" href="#">Characters and Games</a></i>
                 &nbsp;
+                <!-- ko if: partyManagerViewModel.loggedIn -->
+                <i class="small underline"><a class="" data-toggle="modal"
+                  data-target="#partyModal" href="#">Create/Join/Edit Party</a></i>
```

Comment:
I don't like that cause then you aren't sure what this actually does. I agree it's not nice looking. Not sure.